docs/source/user.rst

```diff
@@ -6,16 +6,51 @@ User
 
 How to install
 ===============
-* ``pip install jupyter_alabaster_theme``
-* Edit your ``conf.py`` file to point to the theme:
+To use this theme on any Sphinx based documentation, follow these steps:
+
+Install the ``jupyter_alabaster_theme`` package using pip:
+
+.. code::
+
+    pip install jupyter_alabaster_theme
+
+Edit your ``conf.py`` file:
+
+1. Set the theme to ``jupyter_alabaster_theme``:
+
+.. code-block:: python
+
+    html_theme = 'jupyter_alabaster_theme'
+
+2. Add ``jupyter_alabaster_theme`` to the list of extensions:
+
+.. code-block:: python
+
+    extensions = [
+       ...
+       'jupyter_alabaster_theme',
+    ]
+
+3. At the bottom of ``conf.py``, if the following block of code is present,
+remove it:
 
 .. code-block:: python
 
-    # At the top
-    from jupyter_alabaster_theme import *
-    init_theme()
-    # ...
-    # Comment out the `html_theme =` line
+    if not on_rtd:
+       # only import and set the theme if we're building docs locally
+       import sphinx_rtd_theme
```

Comment:
Is there a reason to use sphinx_rtd_theme? If building locally, you can still use jupyter_alabaster_theme.